packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -73,18 +75,47 @@ const ruleset: any = {
         function: longRunningResponseStatusCodeArm,
       },
     },
+
     // RPC Code: RPC-Async-V1-02
-    ProvisioningStateSpecified: {
-      description: 'A LRO PUT and PATCH operations response schema must have "ProvisioningState" property specified.',
+    //PUT
+    ProvisioningStateSpecifiedForLROPut: {
+      description: 'A LRO PUT operations response schema must have "ProvisioningState" property specified.',
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*[put][?(@property === 'x-ms-long-running-operation' && @ === true)]^"],
+      then: {
+        function: provisioningStateSpecifiedForLROPut,
+      },
+    },
+    //Patch
+    ProvisioningStateSpecifiedForLROPatch: {
+      description: 'A LRO PATCH operations 200 response schema must have "ProvisioningState" property specified.',
```

Comment:
>A LRO PATCH operations 200 response schema must have "ProvisioningState" property specified. [](http://example.com/codeflow?start=20&length=92)

A long running Patch operation response schema must have "ProvisioningState" property specified for the 200 status codes.